docs/cristin.ttl

```diff
@@ -0,0 +1,299 @@
+@prefix rdf:    <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix rdfs:   <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix xsd:    <http://www.w3.org/2001/XMLSchema#> .
+@prefix :       <http://www.example.com/cristin-ontology#> .
+
+#Info about rdfs types
+#https://www.w3.org/TR/rdf-schema/#ch_resource
+
+:Result
+    a            rdfs:Class ;
+    rdfs:comment "The equivalent of an NVA Resource in Cristin. An entry as being exported especially for importing in NVA" .
+
+:PublicationStatus
+    a rdfs:Class .
+
+:Published
+    a               rdfs:Class ;
+    rdfs:label      "PUBLISHED" ;
+    rdfs:subClassOf :PublicationStatus .
+
+:VARBEID_SPRAK_Class
```

Comment:
If some classes or properties are in capital letters, that is because the respective fields are in capital letters. All properties are written the exact same way as they appear in the json file.